.github/workflows/checks.yml

```
@@ -38,3 +38,39 @@ jobs:
 
             - name: Check formatting
               run: yarn format:check
+
+    tuf_on_ci_signing:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)